ming_sim/session.py

```
@@ -1837,6 +1905,84 @@ def apply_cli_conversation_actions(
+                    _extract_imperative_assignee,
+                    _secret_metadata_from_command,
+                )
+                secret_new = [
+                    p for p in confirm_targets
+                    if (
+                        p.get("kind") == "secret_order"
+                        and str(p.get("action") or "") == "新建"
+                    )
+                ]
+                if not secret_new:
+                    # 非密令「修改」：不提前 return，放行既有 directive/office 等补充路径。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)